File: test/src/com/zbiti/spark/test/AcTest.scala
package com.zbiti.spark.test

import com.zb.ac.AC2Util

//import com.zb.ac.AC2Util

//import com.zb.ac.AC2Util
//import com.zb.ac.AC2Util
//import com.zb.ac.AC2Util
//import com.zb.ac.AC2Util


object AcTest {
  def main(args: Array[String]): Unit = {
    var url : String="www.baidu.comsina.com"
    new AC2Util("G:/tmp/jptest/tag_conf.txt", ",")
    val info= AC2Util.getTag(url)
    println(info)
  }
}

File: test/src/com/zb/ac/ACtest.java
package com.zb.ac;

public class ACtest {
	public static void main(String[] args) {
		String url="m.sina.com";
		new AC2Util("/G:/tag_test.txt", ",");
		String tag=AC2Util.getTag(url);
		System.out.println(tag);
	}
}
